operators/test/e2e/framework/builder.go

```diff
@@ -0,0 +1,22 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package framework
```

Comment:
Could this just be named `test`? 

Then we could also have: `test.Step`, `test.StepList`, `test.Run`, `test.Eventually` etc. and could remove the word `Test` from all the methods